packs/slack/sensors/slack_sensor.py

```diff
@@ -237,3 +245,27 @@ def _api_call(self, method, **kwargs):
         result = self._client.api_call(method, **kwargs)
         result = json.loads(result)
         return result
+
+    def _handle_message_channel_join(self, data):
+        trigger = 'slack.message_channel_join'
+        user_info = self._get_user_info(user_id=data['user'])
+
+        payload = {
+            'user': {
+                'id': user_info['id'],
+                'name': user_info['name'],
+                'first_name': user_info['profile'].get('first_name',
+                                                      'Unknown'),
+                'last_name': user_info['profile'].get('last_name',
+                                                      'Unknown'),
+                'real_name': user_info['profile'].get('real_name',
+                                                      'Unknown'),
+                'is_admin': user_info['is_admin'],
+                'is_owner': user_info['is_owner']
+            },
+            'timestamp': int(float(data['ts'])),
+            'timestamp_raw': data['ts'],
+            'text': text
```

Comment:
No, there will still be text for this. We don't actually get `channel` for this event... only a text with `<UID|bob> has joined #channel`. 

WCYD?